test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1

```diff
@@ -329,4 +329,47 @@ Describe "Set/New/Remove-Service cmdlet tests" -Tags "Feature", "RequireAdminOnW
         }
         { & $cmdlet @parameters } | Should -Throw -ErrorId $errorid
     }
+
+    It "Set-Service can run -Status Stopped to stop a service with dependencies, and can use -Force to stop a service with running dependent services" {
+        $testservicename1 = "testservice1"
```

Comment:
The testservice setup should be in a BeforeAll{} block and the cleanup in a AfterAll{} block as I expect future tests may use the test service